package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@celonis/content-cli",
-  "version": "0.10.0",
+  "version": "0.10.1",
```

Comment:
Can you please increase the minor version to 11, because I just merged [a PR](https://github.com/celonis/content-cli/pull/189) that sets the version to `0.11.0`?